diff --git a/dist/components/File.js b/dist/components/File.js
index fa9a91568ab6464890d4570465c7bb1b93828555..24a48dc3617e18884b08d2896939e9e48056ff8e 100644
--- a/dist/components/File.js
+++ b/dist/components/File.js
@@ -945,7 +945,7 @@ var File = class {
 	}
 };
 function shouldRenderCode(pre, file, collapsed = false) {
-	return !collapsed && pre == null && file != null;
+	return !collapsed && (pre == null || !pre.hasChildNodes()) && file != null;
 }
 function shouldRenderHeader(headerElement, file, disableFileHeader = false) {
 	return headerElement == null && file != null && !disableFileHeader;
diff --git a/dist/components/FileDiff.js b/dist/components/FileDiff.js
index 4ee3d95f21fa52834ebd4e726d88669a90722dbd..470024f27ae50cd84df6892634abfd23cac5a04f 100644
--- a/dist/components/FileDiff.js
+++ b/dist/components/FileDiff.js
@@ -1637,7 +1637,7 @@ function hasDiffHeaderContent({ fileDiff, oldFile, newFile }) {
 	return fileDiff != null || oldFile != null || newFile != null;
 }
 function shouldRenderCode(pre, hasContent, collapsed = false) {
-	return !collapsed && pre == null && hasContent;
+	return !collapsed && (pre == null || !pre.hasChildNodes()) && hasContent;
 }
 function shouldRenderHeader(headerElement, hasContent, disableFileHeader = false) {
 	return headerElement == null && hasContent && !disableFileHeader;
